Automattic/simplenote-macos
# Add Sync Indicator to Tags List Footer

The goal of this PR is to help users get a better understanding of how their notes are doing related to sync. If all is well the status indicator will look like this:

<img width="209" alt="screen shot 2019-02-18 at 3 59 18 pm" src="https://user-images.githubusercontent.com/789137/52981858-35f77980-3397-11e9-9ce5-1b4813b31301.png">

If some notes haven't been synced, the UI will update to display the number of unsynced changes. If you click on the sync indicator, it will display which notes haven't synced, along with some help messaging:

<img width="219" alt="screen shot 2019-02-18 at 4 11 06 pm" src="https://user-images.githubusercontent.com/789137/52981953-e9606e00-3397-11e9-8817-2e16322f43d3.png">

**To Test**
* If all notes have synced, the indicator should display `All changes synced`, and clicking on it should show a popover with the last sync date.
* Take the app offline, `No connection` should be displayed in the sync indicator area.
* Make some offline changes. For each note you change, the `Unsynced notes` indicator count should go up.
* Go back online, the notes should sync and the status should go back to `All changes synced`.

p.s. UX feedback welcome! It'll be hard to discover that you can click on the sync indicator to show more info.
